include/swift/AST/DiagnosticsDriver.def

```diff
@@ -114,6 +114,9 @@ ERROR(error_sdk_too_old,none,
 ERROR(error_ios_maximum_deployment_32,none,
       "iOS %0 does not support 32-bit programs", (unsigned))
 
+ERROR(error_arclite_not_found_when_link_objc_runtime,none,
+"unable to find the 'arclite' library", ())
```

Comment:
How about "Unable to find Objective-C runtime support library 'arclite'"? A little wordy, but hopefully explains a bit more to people who run into it unexpectedly.

Reply:
Also, nit: Indent the second and subsequent lines of these diagnostic definitions to match the first.

Reply:
@jrose-apple I think this one is good 
About the `-no-link-objc-runtime` flag, I think could be good mention it :))